cloud/configure-user-authentication-for-clusters.yml

```diff
@@ -0,0 +1,30 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - ../concent-builder/consts.yml
+    - "{{ deployment_values }}/var.yml"
+  tasks:
+    - block:
+        - name: Create a directory for secrets
+          file:
+            path:  "{{ data_dir }}/cloud-secrets/"
+            state: directory
+
+        - name: Upload decrypted "{{ gke.service_account_name }}-private-key.json" secret for activate google cloud service account
+          copy:
+            src:  "{{ local_secret_dir }}/cloud/{{ gke.service_account_name }}-private-key.json"
+            dest: "{{ data_dir }}/cloud-secrets/{{ gke.service_account_name }}-private-key.json"
+            mode: 0444
```

Comment:
Let's upload the key to user's home directory. No need to put it in `{{ data_dir }}` if we're not storing it there permanently.

Reply:
Do you assume that home directory always exists ?